docs/modules/ROOT/pages/howtos/release.adoc

```diff
@@ -14,3 +15,4 @@
 * Tag the merged commit (e.g. `v0.29.0`)
 * Push the tag
 * Create a new https://github.com/camptocamp/devops-stack/releases[release] via the GitHub release page, naming it the same as the version number - e.g. `v0.29.0`. Ensure to state what's new in the release
+* Set back the release to `latest` in https://github.com/camptocamp/devops-stack/blob/master/docs/antora.yml#L4[docs/antora.yml]
```

Comment:
@mcanevet or do we want to simply point to the latest release?

Reply:
I don't know...